app/com/linkedin/drelephant/tuning/Schduler/AzkabanJobStatusManager.java

```diff
@@ -71,12 +76,21 @@ private void assignAzkabanJobStatusUtil() {
   private boolean analyzeJobExecution(JobExecution jobExecution,JobSuggestedParamSet jobSuggestedParamSet){
     try {
       logger.debug(" Getting jobs for Flow "+jobExecution.flowExecution.flowExecId);
+      Pattern jobNamePattern=Pattern.compile("job=([^&]*)");
```

Comment:
Precompile the pattern and have it as a static class member variable

Reply:
Done.